pkg/systemd/services/timer-dialog.jsx

```diff
@@ -109,9 +110,19 @@ const CreateTimerDialogBody = ({ setIsOpen, owner }) => {
             return false;
 
         setInProgress(true);
-        create_timer({ name, description, command, delay, delayUnit, delayNumber, repeat, specificTime, repeatPatterns, owner })
-                .then(() => setIsOpen(false), exc => {
-                    setDialogError(exc.message);
+
+        // Verify if the command exists
+        const command_parts = command.split(" ");
+        cockpit.spawn(["test", "-f", command_parts[0]], { err: "ignore" })
```

Comment:
Hmm, this assumes absolute path, right?

Reply:
Yes, but I don't think ExecStart supports an relative paths? Reading the man page

https://www.man7.org/linux/man-pages/man5/systemd.service.5.html at ExecStart=
```
         For each of the specified commands, the first argument must
           be either an absolute path to an executable or a simple file
           name without any slashes.
           ```
But as far as I know it does not work with an relative path?

```
[jelle@t14s][/usr/bin]%sudo systemd-run -t ExecStart="uname"
Failed to find executable ExecStart=uname: No such file or directory
```

Reply:
Ah right, good to know!